13-location.Rmd

```diff
@@ -498,7 +498,7 @@ reclass = dropLayer(reclass, "pop")
 ```
 
 In common with other data science projects, data retrieval and 'tidying' have consumed much of the overall workload so far.
-With clean data the final step, calculating a final score by summing up all raster layers, can be accomplished in a single line.
+With clean data the final step --- calculating a final score by summing up all raster layers --- can be accomplished in a single line.
```

Comment:
@Robinlovelace pls check if the sentence is clear and understandable, the copyeditor suggested to change it to:

`With clean data as the final step, calculating a final score by summing up all raster layers can be accomplished in a single line.`
Which is clearly not, what I meant.

Reply:
Looks good except I think there should be an additional comma after 'With the data clean', and removal of the the word 'up' which I think adds little/nothing to the meaning, e.g.:

```
With clean data, the final step --- calculating a final score by summing all raster layers --- can be accomplished in a single line of code.
```

And `line of code` is clearer than `line`.


 Good use of ---!